src/cbapi/psc/models.py

```diff
@@ -105,14 +107,26 @@ def _refresh_if_needed(self, request_ret):
 
 
 class Device(PSCMutableModel):
-    urlobject = "/integrationServices/v3/device"
-    primary_key = "deviceId"
-    info_key = "deviceInfo"
+    urlobject = "/appservices/v6/orgs/{0}/devices"
+    urlobject_single = "/appservices/v6/orgs/{0}/devices/{1}"
+    primary_key = "id"
+    #info_key = "deviceInfo"
     swagger_meta_file = "psc/models/deviceInfo.yaml"
```

Comment:
Why deviceInfo instead of just device?

Reply:
I preserved the name of the original file when I copied it from CB Defense.

Reply:
We have the option to change it now so lets make in clearer.

Reply:
This change was made.